pippo-core/src/main/java/ro/pippo/core/route/RouteGroupHandler.java

```diff
@@ -0,0 +1,134 @@
+package ro.pippo.core.route;
+
+import ro.pippo.core.PippoRuntimeException;
+import ro.pippo.core.util.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author ScienJus
+ * @date 16/2/9.
+ */
+public abstract class RouteGroupHandler {
+
+    private String namespace;
+
+    private List<Route> routes = new ArrayList<>();
+
+    public abstract void handle();
```

Comment:
This should probably be `onInit` to match `Application.onInit` - `handle` has a different meaning in the rest of the Pippo architecture.